addons/sale/models/res_config_settings.py

```diff
@@ -60,7 +60,7 @@ class ResConfigSettings(models.TransientModel):
         domain="[('type', '=', 'service')]",
         oldname='deposit_product_id_setting',
         help='Default product used for payment advances')
-    auto_done_setting = fields.Boolean("Lock Confirmed Orders")
+    auto_done_setting = fields.Boolean("Auto Done Setting")
```

Comment:
why this renaming, I thought this task was only disambiguation?

Reply:
Plus it is very confusing now.